python-client/giskard/ml_worker/testing/performance_tests.py

```diff
@@ -572,8 +572,8 @@ def test_diff_reference_actual_rmse(self, reference_slice, actual_slice, model,
               Slice of actual dataset
             reference_slice(Dataset):
               Slice of reference dataset
-            model(Model):
-              Model used to compute the test
+            model(BaseModel):
+              BaseModel used to compute the test
```

Comment:
same comment as for the rest of the "*_tests.py" where "Model used" meant just "Model used", not "BaseModel used"